pkgs/servers/matrix-synapse/default.nix

```diff
@@ -20,45 +20,45 @@ let
       sha256 = "0ss7ld3bpmqm8wcs64q1kb7vxlpmwk9lsgq0mh21a9izyfc7jb2l";
     };
 
-    propagatedBuildInputs = with pythonPackages; [ service-identity ldap3 twisted ];
+    propagatedBuildInputs = with python2Packages; [ service-identity ldap3 twisted ];
 
-    checkInputs = with pythonPackages; [ ldaptor mock ];
+    checkInputs = with python2Packages; [ ldaptor mock ];
   };
-in pythonPackages.buildPythonApplication rec {
+in python2Packages.buildPythonApplication rec {
   name = "matrix-synapse-${version}";
-  version = "0.33.0";
+  version = "0.30.0";
 
   src = fetchFromGitHub {
     owner = "matrix-org";
     repo = "synapse";
     rev = "v${version}";
-    sha256 = "1immk6k0wgiks1s39dhyjg79n6rgans9zy85r5wmkp4dlc3r5rx6";
+    sha256 = "10ggv7669ngxs8py82k8z24874ga0ldcxvpp7xhjpbr1s0gw8zv8";
   };
 
   patches = [
     ./matrix-synapse.patch
   ];
 
-  propagatedBuildInputs = with pythonPackages; [
+  propagatedBuildInputs = with python2Packages; [
     blist canonicaljson daemonize dateutil frozendict pillow pyasn1
     pydenticon pymacaroons-pynacl pynacl pyopenssl pysaml2 pytz requests
-    signedjson systemd twisted ujson unpaddedbase64 pyyaml prometheus_client
```

Comment:
I think `prometheus_client` was added on purpose: https://github.com/matrix-org/synapse/blob/1241156c82644d5609f45659607a356af5d8fe08/synapse/python_dependencies.py#L59
It's an optional dependecny however. Is there any policy regarding these, @FRidh?

Reply:
It's up to the maintainers to decide what to include and what not.

Reply:
Then we shouldn't remove it.